# long-reads_pipelines
##
# Evaluation of current bioinformatics pipelines for analysis of long-read transcriptomic data
##
# Cranfield University
##
# This study explores the benefits of long-read technologies for transcriptomic sequencing, focusing on bioinformatics tools and algorithms for data analysis. Long-read sequencing platforms, such as Nanopore and PacBio, excel in capturing full-length transcripts and accurately identify complex isoforms and alternative splicing events. The study highlights the significance of quality control (QC) and preprocessing tools designed for long-read data, such as NanoPack and Chopper, which prepare data for downstream analysis by addressing specific quality issues associated with these technologies. Using the Oxford Nanoporeâ€™s wf-transcriptomes pipeline, the project demonstrates an effective approach to combining multiple tasks into a single workflow, starting from alignment, through transcript assembly and annotation, to fusion genes detection. A comparative analysis of isoform identification tools, including StringTie and ISOQuant, reveals the significant discordance between their results: while StringTie prioritises alignment with existing reference transcriptome, ISOQuant excels in discovering the novel isoforms. Fusion gene detection is also markedly improved with long-read sequencing, as exemplified by JAFFAL's identification of high-confidence fusion events. The study underscores that long-read sequencing technologies offer more detailed and accurate insights into transcriptome complexity compared to short-read approaches. Anticipated advancements in bioinformatics tools and technologies are expected to enhance the accuracy of transcriptomic analyses further, leading to deeper understanding and exploration of gene expression, fusions and isoform diversity.
